failures
debug_callableNotUnique
| code/class_properties.py:7:5:7:18 | Function arg | Qualified function name 'Prop.arg' is not unique. Please fix. |
| code/class_properties.py:12:5:12:25 | Function arg | Qualified function name 'Prop.arg' is not unique. Please fix. |
| code/class_properties.py:17:5:17:18 | Function arg | Qualified function name 'Prop.arg' is not unique. Please fix. |
